Backend/mentii/user_ctrl.py

```diff
@@ -345,6 +346,21 @@ def joinClass(jsonData, dynamoDBInstance, email=None):
         response.addToPayload('code', updatedClass['code'])
   return response
 
+def leaveClass(jsonData, dynamoDBInstance, email=None):
```

Comment:
This method seems unnecessary since it pretty much does the same thing as removeStudent in class_ctrl?